selfdrive/modeld/parse_model_outputs.py

```diff
@@ -84,23 +84,27 @@ def parse_mdn(self, name, outs, in_N=0, out_N=1, out_shape=None):
     outs[name] = pred_mu_final.reshape(final_shape)
     outs[name + '_stds'] = pred_std_final.reshape(final_shape)
 
-  def parse_outputs(self, outs: dict[str, np.ndarray]) -> dict[str, np.ndarray]:
+  def parse_vision_outputs(self, outs: dict[str, np.ndarray]) -> dict[str, np.ndarray]:
```

Comment:
Does this cause issues in xx? Parse outputs is used there too?

Reply:
nope, xx has a different/independent parser

Reply:
yeah but should be easy to fix